Mathlib/Analysis/Analytic/IsolatedZeros.lean

```diff
@@ -240,6 +240,46 @@ lemma apply_eq_zero_of_order_toNat_ne_zero (hf : AnalyticAt 𝕜 f z₀) :
   simp [hf.order_eq_zero_iff]
   tauto
 
+/-- Helper lemma for `AnalyticAt.order_mul` -/
+lemma order_of_locallyZero_mul_analytic {f g : 𝕜 → 𝕜} (hf : AnalyticAt 𝕜 f z₀)
+  (hg : AnalyticAt 𝕜 g z₀) (h'f : hf.order = ⊤) :
+    (hf.mul hg).order = ⊤ := by
+  rw [AnalyticAt.order_eq_top_iff, eventually_nhds_iff] at *
+  obtain ⟨t, h₁t, h₂t, h₃t⟩ := h'f
+  exact ⟨t, fun y hy ↦ (by rw [h₁t y hy]; ring), h₂t, h₃t⟩
+
+/-- The order is additive when multiplying analytic functions -/
```

Comment:
Full stop, please: this is a complete sentence

Reply:
Done, here and elsewhere.